titiler/pgstac/main.py

```diff
@@ -29,14 +29,15 @@
 logging.getLogger("rio-tiler").setLevel(logging.ERROR)
 
 settings = ApiSettings()
+pg_settings = PostgresSettings()
 
 app = FastAPI(title=settings.name, version=titiler_pgstac_version)
 
 
 @app.on_event("startup")
 async def startup_event() -> None:
     """Connect to database on startup."""
-    await connect_to_db(app)
+    await connect_to_db(app, settings=pg_settings)
```

Comment:
@alukach do we still want this? with your latest commit we could remove passing the settings here?

Reply:
Sure, to keep things simple we can default to having settings initialized in the `connect_to_db()` func